cmd/helmvm/upgrade.go

```diff
@@ -148,21 +148,9 @@ var upgradeCommand = &cli.Command{
 			metrics.ReportNodeUpgradeFailed(c.Context, err)
 			return err
 		}
-		os.Setenv("KUBECONFIG", kcfg)
-		logrus.Infof("Upgrading addons")
-		opts := []addons.Option{}
-		if c.Bool("no-prompt") {
-			opts = append(opts, addons.WithoutPrompt())
-		}
-		for _, addon := range c.StringSlice("disable-addon") {
-			opts = append(opts, addons.WithoutAddon(addon))
-		}
-		if err := addons.NewApplier(opts...).Apply(c.Context); err != nil {
-			err := fmt.Errorf("unable to apply addons: %w", err)
-			metrics.ReportNodeUpgradeFailed(c.Context, err)
-			return err
-		}
+
```

Comment:
We were applying the new add-ons but now we are not. Shouldn't we be updating the k0s configuration with the new add-ons here ?